pkg/ui/workspaces/db-console/src/redux/tenants.ts

```diff
@@ -38,6 +43,17 @@ export const isSystemTenant = (tenantName: string): boolean => {
   return tenantName === SYSTEM_TENANT_NAME;
 };
 
+export const containsApplicationTenants = (
+  tenantOptions: DropdownOption[],
+): boolean => {
+  return (
+    tenantOptions.filter(
```

Comment:
nit: You can also accomplish this with tenantOptions.some( t => t.label !== SYSTEM_TENANT_NAME && t.label !== ALL_TENANTS_OPTION.label) which won't have to operate on the entire array / create a new one.

Feel free to ignore though as this array is likely always quite small anyway.

Reply:
Thx! Done.